numismatic/feeds/base.py

```diff
@@ -39,7 +39,6 @@ class Subscription:
     symbol = attr.ib()
     channel = attr.ib()
     client = attr.ib()
-    listener = attr.ib()
```

Comment:
We don't need the `listener` attribute for the Websocket subscriptions anymore but this obviously breaks the REST subscriptions for now.

Reply:
Fixed in latest commit